mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -382,120 +421,254 @@ def StartVisualizePointCloudTask(self, vminitparams, locationName=None, sensorSe
+
+        Returns:
+            dict: a list of the latest detection results, having the structure
+            A dictionary with the structure:
+
+            detectionResults (dict): A dictionary with the structure:
+
+                cycleIndex (str): Unique cycle index string for tracking, backing up, and differentiating cycles.
+                detectedObjects (list)
+                detectionResultState (dict)
+                imageEndTimeStampMS (int)
+                imageStartTimestampMS (int)
+                locationName (str)
+                pointCloudId (str)
+                resultTimestampMS (int)
+                sensorSelectionInfos (list)
+                statsUID (str)
+                targetUpdateName (str)
+                taskId (str)
         """
-        command = {'command': 'GetLatestDetectedObjects'}
-        if taskId:
+        command = {
+            'command': 'GetLatestDetectedObjects',
+        }
+        if taskId is not None:
             command['taskId'] = taskId
-        if cycleIndex:
+        if cycleIndex is not None:
             command['cycleIndex'] = cycleIndex
-        if taskType:
+        if taskType is not None:
             command['taskType'] = taskType
         return self._ExecuteCommand(command, timeout=timeout)
-    
+
     def GetLatestDetectionResultImages(self, taskId=None, cycleIndex=None, taskType=None, newerThanResultTimestampMS=0, sensorSelectionInfo=None, metadataOnly=False, imageTypes=None, limit=None, blockwait=True, timeout=2.0):
-        """gets the latest detected objects
+        """Gets the latest detected result images.
+
+        Args:
+            taskId (str, optional): If specified, the taskId.
+            cycleIndex (str, optional): Unique cycle index string for tracking, backing up, and differentiating cycles.
+            taskType (str, optional): The task type.
+            newerThanResultTimestampMS (int, optional): If specified, starttimestamp of the image must be newer than this value in milliseconds. (Default: 0)
+            sensorSelectionInfos:
+            metadataOnly (bool, optional): Default: False
+            imageTypes (list, optional): Mujin image types
+            limit (int, optional):
+            blockwait: (Default: True)
+            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 2.0)
+
+        Returns:
+            str: Raw image data
         """
-        log.verbose("Getting latest detection result images...")
-        command = {'command': 'GetLatestDetectionResultImages'}
-        if newerThanResultTimestampMS:
-            command['newerThanResultTimestampMS'] = newerThanResultTimestampMS
-        if taskId:
+        command = {
+            'command': 'GetLatestDetectionResultImages',
+            'newerThanResultTimestampMS': newerThanResultTimestampMS,
+            'metadataOnly': metadataOnly,
```

Comment:
This is the only section that changes client behavior, but it should not result in any system behavior change. The defaults set here are the same as in the server.